### PR TITLE
export new implemented oltp traversals

### PR DESCRIPTION
**Pull request by zhoney**
Change-Id: I0e501dcffd7c38dbd6afe76e82000935160bee36

### CODE REVIEW

File: hugegraph-dist/src/assembly/static/conf/gremlin-server.yaml [hunk 1]
```diff
@@ -20,15 +20,27 @@ scriptEngines: {
           com.baidu.hugegraph.backend.id.IdGenerator,
           com.baidu.hugegraph.type.define.Directions,
           com.baidu.hugegraph.type.define.NodeRole,
-          com.baidu.hugegraph.traversal.algorithm.CustomizePathsTraverser,
+          com.baidu.hugegraph.traversal.algorithm.CollectionPathsTraverser,
```
Comment (javeme) [inline hugegraph-dist/src/assembly/static/conf/gremlin-server.yaml:23]: export EdgeStep

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/TraverserAPI.java [hunk 2]
```diff
@@ -38,14 +38,11 @@ protected static EdgeStep step(HugeGraph graph, Step step) {
...
-    protected static TemplatePathsTraverser.RepeatEdgeStep repeatEdgeStep(
-              HugeGraph graph, TemplatePathsAPI.RepeatEdgeStep step) {
-        return new TemplatePathsTraverser.RepeatEdgeStep(graph, step.direction,
-                                                         step.labels,
-                                                         step.properties,
-                                                         step.degree,
-                                                         step.skipDegree,
-                                                         step.maxTimes);
+    protected static RepeatEdgeStep repeatEdgeStep(
+              HugeGraph graph, TemplatePathsAPI.TemplatePathStep step) {
```
Comment (javeme) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/TraverserAPI.java:42]: move to TemplatePathsAPI

**Review by javeme — CHANGES_REQUESTED** [review]

File: hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/CountTraverser.java [hunk 3]
```diff
@@ -63,45 +62,45 @@ public long count(Id source, List<Step> steps,
...
-            long edgesCount = this.edgesCount(target, lastStep.edgeStep);
+            long edgesCount = this.edgesCount(target, lastStep);
             this.count.add(edgesCount);
         }
 
         return this.count.longValue();
     }
 
-    private Iterator<Edge> edgesOfVertex(Id source, Step step) {
+    private Iterator<Edge> edgesOfVertexWithDedup(Id source, EdgeStep step) {
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/CountTraverser.java:99]: edgesOfVertexWithCount

File: hugegraph-dist/src/assembly/static/conf/gremlin-server.yaml (hunk 1 above)
Comment (zhoney) [reply hugegraph-dist/src/assembly/static/conf/gremlin-server.yaml:23]: already export

**Review by javeme — APPROVED** [review]

**Review by Linary — APPROVED** [review]

**Comment by dongma** [comment]
gremlin出发oltp，能否提供一个gremlin语句触发的示例？

**Pull request merged**